### PR TITLE
Colored mentions

### PR DESCRIPTION
**Pull request by KentoNishi**
the approach i took for this is (hopefully) more efficient than what youtube does, since it highlights the entire message rather than each instance of a ping. this also fixes an issue in youtube where pinging someone with an emoji in their name doesn't work.

image

image

### CODE REVIEW

**Comment by KentoNishi** [comment]
image

**Comment by KentoNishi** [comment]
image

now works on vods

**Review by r2dev2 — CHANGES_REQUESTED** [review]
Can you make a settings option to opt into enabling user highlighting?

**Review by r2dev2 — APPROVED** [review]

**Comment by KentoNishi** [comment]
@ChrRubin review pls

File: src/scripts/chat-background.ts [hunk 1]
```diff
@@ -428,7 +436,8 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
   } else if (request.type === 'createPopup') {
     chrome.windows.create({
       url: request.url,
-      type: 'popup'
+      type: 'popup',
+      state: 'maximized'
```
Comment (ChrRubin) [inline src/scripts/chat-background.ts:440]: Wait why are we maximizing popups? This will affect the LiveTL popup too.

Comment (KentoNishi) [reply src/scripts/chat-background.ts:440]: I tested the pop-up window on multiple screen sizes, and for some reason it keeps opening a super vertically short window (maybe like 20px tall 400px wide). Idk maybe this is just a me problem, I guess I can just remove this part then

**Review by ChrRubin — APPROVED** [review]

**Pull request merged**